src/Explorer/Panes/DeleteCollectionConfirmationPaneComponent.tsx

```diff
@@ -0,0 +1,201 @@
+import * as NotificationConsoleUtils from "../../Utils/NotificationConsoleUtils";
+import * as TelemetryProcessor from "../../Shared/Telemetry/TelemetryProcessor";
+import * as React from "react";
+import { Action, ActionModifiers } from "../../Shared/Telemetry/TelemetryConstants";
+import { PanelFooterComponent } from "./PanelFooterComponent";
+import { Collection } from "../../Contracts/ViewModels";
+import { Text, TextField } from "office-ui-fabric-react";
+import { userContext } from "../../UserContext";
+import { Areas } from "../../Common/Constants";
+import { DefaultAccountExperienceType } from "../../DefaultAccountExperienceType";
+import { CassandraAPIDataClient } from "../Tables/TableDataClient";
+import { deleteCollection } from "../../Common/dataAccess/deleteCollection";
+import { getErrorMessage, getErrorStack } from "../../Common/ErrorHandlingUtils";
+import { DefaultExperienceUtility } from "../../Shared/DefaultExperienceUtility";
+import { PanelErrorComponent, PanelErrorProps } from "./PanelErrorComponent";
+import DeleteFeedback from "../../Common/DeleteFeedback";
+import Explorer from "../Explorer";
+import LoadingIndicator_3Squares from "../../../images/LoadingIndicator_3Squares.gif";
+
+export interface DeleteCollectionConfirmationPaneProps {
+  explorer: Explorer;
```

Comment:
This is not ideal. Would like to avoid coupling more stuff to the entire Explorer class. But I don't see any way around right now. Maybe once we extract resource tree....

Reply:
Yeah I actually tried to remove this dependency but it turned out to be a lot of work so I'm going remove it in a separate PR.